### ISSUE TITLE
does not work on .net 3.5 full and client profile

### ISSUE THREAD

**Issue by berkantipek** [justcoding121/Titanium-Web-Proxy#378]
Install-Package : Could not install package 'StreamExtended 1.0.81'. You are trying to install this package into a pro
ject that targets '.NETFramework,Version=v3.5,Profile=Client', but the package does not contain any assembly reference
s or content files that are compatible with that framework. For more information, contact the package author.

**Comment by honfika** [comment]
StreamExtended (and TWP) is using async methods which is available in .NET 4.5. It won't work in older .NET frameworks.
Which method do you need from StreemExtended. I think it would be easier just copy the required source code to your project, StreamExtended is quite small.

You can find the source here: https://www.nuget.org/packages/StreamExtended/

**Comment by berkantipek** [comment]
@honfika I'm not using any direct method from StreamExtended library. I am using ProxyServer and ExplicitProxyEndPoint classes from TWP.

**Comment by honfika** [comment]
TWP was designed to use async methods, it is impossbile to use it in .NET 3.5, I'm sorry.
@justcoding121 Is the owner of this package, but I'm sure that he would answer the same

Why do you need that old .NET version? It was released in 2007 and the mainstream support of 3.5 SP1 was ended more than 3 years ago. (ok, the extended support is still valid)

**Comment by berkantipek** [comment]
@honfika I published an app written with C# and it crashes in some people's computers. I don't know why. I am not able to get a debug info or crash details from users so I decided to ship it with .Net FW 3.5 instead of 4.5. If I'm not wrong, version 4.5 is shipped after Windows 7 but I don't actually know in detail. Maybe it is because that I pack and secure the .Net assembly and required DLLs with ConfuserEx.
